packages/frontity-org-theme/src/processors/homepage-hero.ts

```diff
@@ -0,0 +1,19 @@
+import { Element } from "@frontity/html2react/himalaya/types";
+import { Processor } from "@frontity/html2react/types";
+import { HTMLProps } from "react";
+
+import HomepageHero from "../components/homepage-hero";
+
+export const homepageHeroAnimation: Processor<HTMLProps<Element>> = {
+  name: "homepage-hero-animation",
+  test: ({ node }) =>
+    node.type === "element" &&
+    node.props.className.split(" ").includes("homepage-hero-animation"),
```

Comment:
Hey, you forgot to add a `?` after `className`. If you look into the console logs you will see a lot of warning messages saying `Cannot read "split" of undefined`.

Reply:
Oh! Thanks, David.